galaxy/templates/deployment-web.yaml

```diff
@@ -45,6 +45,34 @@ spec:
             - name: galaxy-data
               mountPath: /galaxy/server/config/mutable/
               subPath: config
+        - name: {{ $.Chart.Name }}-initscripts
+          image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag }}"
+          command: ['sh', '-c', 'echo "Starting to run init scripts"
+          {{- range $key, $entry := .Values.initScripts -}}
+          {{- if $entry.applyToWeb -}}
+           && echo "Running {{$key}}.sh" && sh /galaxy/initscripts/{{$key}}.sh
+          {{- end -}}
+          {{- end -}}
+          ']
+          volumeMounts:
+            {{- range $key, $entry := .Values.initScripts -}}
+            {{- if $entry.applyToWeb }}
+            - name: galaxy-initscripts-{{ $key | lower }}
+              mountPath: /galaxy/initscripts/{{$key}}.sh
+              subPath: {{$key}}.sh
+            {{- end }}
+            {{- end }}
+            - name: galaxy-data
+              mountPath: {{ $.Values.persistence.mountPath }}
+        {{- range $key, $entry := .Values.initContainers -}}
+        {{- if $entry.applyToWeb }}
+        - name: {{ $.Chart.Name}}-{{ $key | lower }}
```

Comment:
Can we include the entire fragment like this?
https://github.com/helm/charts/blob/e35e046cf78c89d800362dd97fbd761d2c892a70/stable/keycloak/templates/statefulset.yaml#L62

Reply:
We can if we remove the `applyTo[Web|Pod]` booleans. But if we want the option to choose which deployment/pod they should be run in, we have to extract the relevant values to ignore those booleans (might be able to still do it more directly if we do a `range` loop and ignore the two keys specifically, maybe? Is that more desirable though?

Reply:
Changed to put them as full init containers in the values files with the special two keys `applyTo[Web|Pod]` that get `unset` in the deployment manifests before rendering. Contents still go through `tpl`.